history/characters/22000_wildhammer.txt

```diff
@@ -43,7 +43,20 @@
 	378.3.25={
 		#trait = class_warrior_7
 	}
-	591.11.1={ death = { death_reason = death_vanished } }
+	591.11.1 = {
+		effect = {
+			if = {
+				limit = {
+					NOT = {
+						current_date >= 609.1.6
+					}
```

Comment:
Not is not needed here.
```suggestion
					current_date < 609.1.6
```

Reply:
Applied all of these suggestions in 4532e4e20bc28c7b8c462332ae64472224db95a4, but I did it manually as the exact date is important in these so I wanted them to have the `<=` sign instead of `<`.